### ISSUE TITLE
Support for rails 5

### ISSUE THREAD

**Issue by haobole** [camertron/jvectormap-rails#4]
It appears that the current gem does not support rails 5. Do you have any intention of supporting it?

```
Bundler could not find compatible versions for gem "railties":
  In Gemfile:
    jvectormap-rails was resolved to 1.0.0, which depends on
      railties (~> 3.1)

    rails (~> 5.0.4) was resolved to 5.0.4, which depends on
      railties (= 5.0.4)
```

**Comment by camertron** [comment]
Hey @haobole, thanks for filing an issue. It turned out to be pretty easy to support Rails 5. I also went ahead and upgraded the packaged version of jvectormap and imported a bunch of additional maps from the jvectormap website. It's all part of the `rails_5` branch.

I created a new dummy rails app and tested things out, but I want to make sure it works for you too. Would you mind testing the changes to make sure they work for you? You should be able to put this in your Gemfile:

```ruby
gem 'jvectormap-rails', github: 'camertron/jvectormap-rails', branch: 'rails_5'
```

Thanks!

**Comment by haobole** [comment]
@camertron , thanks for your responsiveness.

After I moved to the branch it, it did not work for me. I get this error:

```
Sprockets::FileNotFound - couldn't find file 'jvectormap/maps/us_aea_en' with type 'application/javascript'
Checked in these paths:
  /Users/philiphallstrom1/workspace/ads/app/assets/fonts
  /Users/philiphallstrom1/workspace/ads/app/assets/images
  /Users/philiphallstrom1/workspace/ads/app/assets/javascripts
  /Users/philiphallstrom1/workspace/ads/app/assets/stylesheets
  /Users/philiphallstrom1/workspace/ads/vendor/assets/javascripts
  /Users/philiphallstrom1/workspace/ads/vendor/assets/stylesheets
```

I already have these two configurations in place as I have been using the old version of jvectormap-rails for 3 years:

config/initializers/jvectormap.rb:

```
JVectorMap::Rails.precompile_maps << "us_aea_en"
```

And these lines in assets/application.js:

```
//= require jvectormap
//= require jvectormap/maps/us_aea_en
```

Any idea? Did I miss something?

**Comment by camertron** [comment]
@haobole ah I think I know what's going on. The rails_5 branch includes updates to the embedded jvectormap library _and_ the maps scraped from the jvectormap website. For some reason the maintainers removed the language suffixes from the map names, so instead of `us_aea_en` it should be `us_aea`. If you run `bundle exec rake jvectormap:maps` you should see a list of all the available maps. I just verified that `us_aea` map is on the list. Let me know if that works :)

**Comment by haobole** [comment]
Thanks, @camertron . I can include it now without errors on the rails. The rendering is all wrong, but I suspect that has to do with changes in vectormap itself. I will investigate those myself.

**Comment by camertron** [comment]
@haobole ok awesome. Published as v2.0.
